azext_iot/central/providers/device_provider.py

```diff
@@ -194,20 +201,92 @@ def get_device_registration_info(
 
         return info
 
-    def get_all_registration_info(self, central_dns_suffix="azureiotcentral.com"):
+    def filter_dps_info(self, dps_info, device_status):
```

Comment:
what is the expected behavior if device_status is None?

Reply:
device_status should have a value of DeviceStatus and in which case all the values for enum are handled. Passing in None will result in get returning NULL which is acceptable.